### PR TITLE
[snap/frontend]: Added account list component

### PR DESCRIPTION
**Pull request by AnthonyLaw**
## What was the issue?
- Display all Symbol accounts from the snap.

## What's the fix?
- Added account list modal box
- click on the profile image, and the modal box pops up.

image

### CODE REVIEW

File: snap/frontend/components/AccountInfo/index.jsx [hunk 1]
```diff
@@ -1,16 +1,28 @@
...
+import AccountListModalBox from '../AccountListModalBox';
 import Address from '../Address';
+import { useState } from 'react';
 
 const AccountInfo = () => {
 	const { walletState } = useWalletContext();
-	const { selectedAccount } = walletState;
+	const { accounts, selectedAccount } = walletState;
+
+	const [ isOpenAccountListModalBox, setOpenAccountListModalBox ] = useState(false);
```
Comment (Wayonb) [inline snap/frontend/components/AccountInfo/index.jsx:10]: isOpenAccountListModalBox => accountListModalBoxVisible?

File: snap/frontend/components/AccountListModalBox/AccountListModalBox.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import AccountListModalBox from '.';
+import testHelper from '../testHelper';
+import { screen } from '@testing-library/react';
+
+describe('components/AccountListModalBox', () => {
+	const generateAccountsState = numberOfAccounts => {
+		const accounts = {};
+
+		for (let index = 0; index < numberOfAccounts; index++) {
```
Comment (Wayonb) [inline snap/frontend/components/AccountListModalBox/AccountListModalBox.spec.jsx:9]: Should this be a helper function to create X number of accounts?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: snap/frontend/components/AccountListModalBox/AccountListModalBox.spec.jsx (hunk 2 above)
Comment (AnthonyLaw) [reply snap/frontend/components/AccountListModalBox/AccountListModalBox.spec.jsx:9]: yeah, sound good to me.

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**